esx_service/vmdk_ops_test.py

```diff
@@ -996,9 +1004,8 @@ def test_vmdkops_on_tenant_vm(self):
         # listVMDK return vol name with the format like vol@datastore
         # result should have two volumes :tenant1_vol2, and  tenant1_vol3
         self.assertEqual(2, len(result))
-        self.assertEqual("tenant1_vol2@"+self.datastore_name, result[0]['Name'])
-        self.assertEqual("tenant1_vol3@"+self.datastore_name, result[1]['Name'])
-
+        volume_names = ["tenant1_vol2@"+self.datastore_name,"tenant1_vol3@"+self.datastore_name]
```

Comment:
please stay with the code style. 

Also, I suggest a function for constructing volume_names - instead of the copy-pasted code used now